packages/terra-clinical-result/tests/jest/ResultTimeHeaderCell.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ResultTimeHeaderCell from '../../src/resulttimeheadercell/ResultTimeHeaderCell';
+
+// Snapshot Tests
+describe('FlowsheetResultCell', () => {
+  it('should render', () => {
+    const cell = shallow(<ResultTimeHeaderCell />);
```

Comment:
should these be using render? or should we update the test to say shallow render?